src/main/java/org/broadinstitute/hellbender/tools/walkers/bqsr/AnalyzeCovariates.java

```diff
@@ -477,13 +477,7 @@ private File resolveCsvFile() {
         if (csvFile != null)  {
             return csvFile;
         } else {
-            try {
-              final File result = File.createTempFile("AnalyzeCovariates", ".csv");
-              result.deleteOnExit();
-              return result;
-            } catch (IOException e) {
-                throw new UserException("Could not create temporary Csv file",e);
-            }
+            return BaseTest.createTempFile("AnalyzeCovariates", ".csv");
```

Comment:
It is very awkward for tools to be calling to a `BaseTest` method -- can we move this to `IOUtils`, have non-test classes call `IOUtils.createTempFile()`, and have `BaseTest.createTempFile()` call `IOUtils.createTempFile()`?